js/main.js

```diff
@@ -0,0 +1,99 @@
+$(document).ready(function() {
+ $.ajax({
+   url: 'http://localhost:3000/articles',
+   type: 'get'
+    dataType: 'json'
+ }).done(ArticleApp.getArticles);
+});
+
+// Create a namespace for my app
+ ArticleApp = ArticleApp || {};
+
+// callback handler for all articles from server
+ArticleApp.getArticles = function(data){
+ //debugger;
+ console.log(data);
+
+ var string = JSON.stringify(data);
```

Comment:
What are you expecting to do here? This would turn your data into a JSON string. How does that fit in?